eng/common/scripts/git-branch-push.ps1

```diff
@@ -40,28 +41,24 @@ param(
 # Without explicitly setting the ErrorActionPreference to continue the script
 # would fail the first time git wrote command output.
 $ErrorActionPreference = "Continue"
+$RemoteName = "azure-sdk-fork"
 
-if (!(git remote | ? {$_ -eq $RemoteName}))
```

Comment:
If the remote exists and doesn't match the GitUrl this script well end up failing. So we should add a check to ensure that the remote name doesn't exists but if it does exist that it matches the GitUrl.

Reply:
Something like:

```
if ((git remote) -contains $RemoteName)
{
  $remoteUrl = git remote get-url $RemoteName
  if ($remoteUrl -ne $GutUrl)
  {
     Write-Error "Remote with name $RemoteName already exists with an incompatible url [$remoteUrl] which should be [$giturl]."
     exit 1
  }
}

```